README.rst

```diff
@@ -828,6 +828,14 @@ This should end with a 'PASSED' result::
     PASSED (successes=143)
 
 
+To install the synapse homeserver from your source run:
+
+    source env/bin/activate
+    mkdir build && cd build
+    pip install ..
+
+This should install synapse in the `/build` directory
```

Comment:
I made a build directory so that it keeps config files & SSL certificates in a separate folder .
I now realised that this could have been done with `--config-path build/*****`
Yes I think we can run synctl directly. I was trying to install it like a release